configuration/pih/petl/jobs/sql/extractions/mch_kellogg_data.sql

```diff
@@ -0,0 +1,311 @@
+-- set @locale = 'en';  -- uncomment for testing
```

Comment:
If the tablename is now "mch_j9_data" (which is much better), then this filename needs to change to "mch_j9_data.sql" as well.